### ISSUE TITLE
Feature Request : Adding field side to Fills and Order model classes

### ISSUE THREAD

**Issue by vivekspanicker** [pcrady/coinbase_exchange#3]
Could you please include 'side' in Fills and Order model classes? This field indicates whether it is  buy/sell fill or order. Thank you for this great work.

```
[
  {
    "created_at": "2019-11-21T01:38:23.878Z",
    "trade_id": 78098253,
    "product_id": "BTC-USD",
    "order_id": "41473628-db2c-464e-b9f4-82df7e4fb4f4",
    "user_id": "5cf6e115aaf44503db300f1e",
    "profile_id": "8058d771-2d88-4f0f-ab6e-299c153d4308",
    "liquidity": "T",
    "price": "8087.38000000",
    "size": "0.00601800",
    "fee": "0.2433492642000000",
    "side": "sell",
    "settled": true,
    "usd_volume": "48.6698528400000000"
  }
]
```

**Comment by pcrady** [comment]
Yeah for sure

**Comment by pcrady** [comment]
I added it to the Fill class but it was already in the Order class. Fixed with version 0.0.6.
